client/protoc-gen-improbable-grpc-web/generate-protos.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)"
+PROTOBUF_DIR=${PROTOBUF_DIR-${SCRIPT_DIR}/proto}
+PROTOGEN_DIR=generated/_proto
+GENERATION_DIR=${GENERATION_DIR-${SCRIPT_DIR}/${PROTOGEN_DIR}}
+
+mkdir -p ${GENERATION_DIR} 2> /dev/null
+PROTO_SOURCES=$(npx glob-cli2 './proto/**/*.proto')
```

Comment:
:scream:

Reply:
Yeah, fair, this script was hacky AF! Your suggestion of prototool is a good one.